physics/GFS_rrtmgp_pre.F90

```diff
@@ -15,16 +15,19 @@ module GFS_rrtmgp_pre
        NF_VGAS,                  & !< Number of active gas species
        getgases,                 & !< Routine to setup trace gases
        getozn                      !< Routine to setup ozone
-  ! RRTMGP types
-  use mo_gas_concentrations, only: ty_gas_concs
   use radiation_tools,       only: check_error_msg,cmp_tlev
+  use rrtmgp_lw_gas_optics,  only: lw_gas_props  
```

Comment:
@dustinswales This reminds me to ask about RRTMGP DDTs, since I've forgotten where this landed when it was implemented. In the past, I think that the we have said that if CCPP schemes use their own DDTs that they be confined to those that only used "internally" in physics. I.e., that in the long run we're not requiring hosts to "know" about more DDTs than necessary. So, in practice, that would mean confining RRTMGP-specific DDTs to the current Interstitial top-level DDT because all of those will technically be able to be allocated by the CCPP in the future with capgen. Is this your understanding and are your changes consistent with this? Am I totally off about this?

Reply:
@grantfirl This PR address the concerns you brought up. Up until now, there were two types of DDTs in the GP scheme: internal DDTs that exist with out the model's knowledge (i.e. lw_gas_props), and interstitial DDT's. For the former we had to add hooks for ccpp compliance in the rte-rrtmgp submodule, and also inform the host about these GP DDTs. This was a headache to manage and is now longer needed with the refactoring.